### PR TITLE
[NUI] Introduce WindowLayoutType enum and SetLayout Method

### PR DESCRIPTION
**Pull request by dkdk-ryu**
The new `SetLayout` method allows users to easily snap windows to specific positions and sizes. This feature is allowing users to quickly arrange their windows in a convenient and efficient manner.

DALi:
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-adaptor/+/290338/
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-csharp-binder/+/290377

### CODE REVIEW

**Comment by sparrow74** [comment]
LGTM :)

**Review by rabbitfor — APPROVED** [review]
👍

File: src/Tizen.NUI/src/public/Window/Window.cs [hunk 1]
```diff
@@ -1752,6 +1752,80 @@ public void SetMimimumSize(Size2D size)
...
+                case WindowLayoutType.TopThird:
+                    Interop.Window.SetLayout(SwigCPtr, 1, 3, 0, 0, 1, 1);
+                    break;
+                case WindowLayoutType.MiddleThird:
+                    Interop.Window.SetLayout(SwigCPtr, 1, 3, 0, 1, 1, 1);
+                    break;
+                case WindowLayoutType.BottomThird:
+                    Interop.Window.SetLayout(SwigCPtr, 1, 3, 0, 2, 1, 1);
+                    break;
+            }
```
[the remark this answers is not in the document]
Comment (dkdk-ryu) [reply src/Tizen.NUI/src/public/Window/Window.cs:1827]: I'll apply this commit to API10.
ThrowExceptionIfExists() doesn't exist in API10. So I changed it to the other method again.

**Comment by dkdk-ryu** [comment]
I'll apply this commit to API10.
ThrowExceptionIfExists() doesn't exist in API10. So I changed it to the other method again.

**Review by huiyueun — APPROVED** [review]

**Pull request merged**